share/picongpu/examples/ThermalTest/include/picongpu/ThermalTestSimulation.hpp

```diff
@@ -197,7 +197,7 @@ namespace picongpu
         //   you may like to let the plasma develope/thermalize a little bit
         static constexpr uint32_t firstTimestep = 1024;
 
-        container::HostBuffer<float, 2>* eField_zt[2];
+        container::HostBuffer<float, 2>* eField_zt[2] = {nullptr, nullptr};
```

Comment:
I think a better solution would be to default-init the `container::HostBuffer` pointer to `nullptr`

Reply:
oh wait, forget what I said. this is a raw pointer to a container here